testcases/testcase_blood_flow_model.py

```diff
@@ -39,6 +39,9 @@
         "ht_constant": 0.3,  # only required if RBC impact is considered
         "mu_plasma": 0.0012,
 
+        # if True set the blood vessel with unrealistic blood flow to zero
+        "low_flow_vessel": True,
```

Comment:
Where and why do you use this?

Reply:
This is the parameter to allow or not the setting of the zero flow vessel approach.
If it is True we going to use this threshold, otherwise (False) we will not

Reply:
I see.. so if it is True, will it set the very low blood flow to zero? 
I was confused by the word "unrealistic". Can you please change the comment accordingly as you described it now?